utils/realtek-poe/Makefile

```diff
@@ -3,14 +3,15 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=realtek-poe
-PKG_RELEASE:=2
+PKG_RELEASE:=1
 PKG_LICENSE:=GPL-2.0
 PKG_MAINTAINER:=Martin Kennedy <hurricos@gmail.com>
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL:=https://github.com/Hurricos/realtek-poe.git
-PKG_SOURCE_VERSION:=ea32075ae41874043eeed14156755a4c0f114582
-PKG_MIRROR_HASH:=5883af61645216eaf90e86a372df055ef352c4445d4f5010da629d98771da0ec
+PKG_SOURCE_VERSION:=7bc6a39f9e242184db793174cf19e784b24fcf3f
+PKG_SOURCE_DATE:=2025-06-17
+PKG_MIRROR_HASH:=895138ee9bd819baa705392e454f3c83c3027c7003cfa8ca37c62b900783f063
```

Comment:
This is still wrong.

CI fails:
  WARNING: PKG_MIRROR_HASH does not match realtek-poe-2025.06.17~7bc6a39f.tar.zst hash a59cdb9c9226d2cb659ce1550715746014a6c88aa56b0466e48c3a6c6d9099cb

Reply:
Will be fixed once you switch to use tagged releases:

```
PKG_NAME:=realtek-poe
PKG_VERSION:=1.2
PKG_RELEASE:=1

PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
PKG_SOURCE_URL:=https://github.com/Hurricos/realtek-poe/archive/refs/tags/v$(PKG_VERSION)
PKG_HASH:=ebe58d2e0f630889d79fb84155936bc43253242857dabfb80d9da71edf92d1e0
```